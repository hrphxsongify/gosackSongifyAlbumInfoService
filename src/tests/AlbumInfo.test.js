import React from 'react';
import { shallow } from 'enzyme';
import AlbumInfo from '../components/AlbumInfo.jsx';

const currentAlbumData = {
  name: '',
  year: 0,
  tracks: []
};

describe('AlbumInfo', () => {
  it('should do some shit', () => {
    const component = shallow(<AlbumInfo albumInfo={currentAlbumData} />);
    expect(component).toMatchSnapshot();
  });
});

// describe('album info tests', () => {
//   it('renders album info', () => {
//     const albumInfo = {
//       imageId: 0,
//       name: 'testies',
//       year: 1942
//     };

//     const wrapper = shallow(<AlbumInfo albumInfo={albumInfo} />);

//     // Expect the wrapper object to be defined
//     expect(wrapper.find('.album-info')).toBeDefined();
//     // expect(wrapper.find('.item')).toHaveLength(items.length);
// //   });
// });
