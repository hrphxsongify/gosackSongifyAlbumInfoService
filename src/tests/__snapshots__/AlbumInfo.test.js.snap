// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlbumInfo should do some shit 1`] = `
<div
  className="album__info"
>
  <div
    className="album__info__art"
  >
    <img
      alt=""
      src=""
    />
  </div>
  <div
    className="album__info__meta"
  >
    <div
      className="album__year"
    >
      0
    </div>
    <div
      className="album__name"
    />
    <div
      className="album__actions"
    >
      <button
        className="button-light save"
      >
        Save
      </button>
      <button
        className="button-light more"
      >
        <i
          className="ion-ios-more"
        />
      </button>
    </div>
  </div>
</div>
`;
